gflownet/utils/buffer.py

```diff
@@ -36,6 +36,11 @@ def __init__(
         self.replay.reward = pd.to_numeric(self.replay.reward)
         self.replay.energy = pd.to_numeric(self.replay.energy)
         self.replay.reward = [-1 for _ in range(self.replay_capacity)]
+        self.replay_states = {}
+        self.replay_trajs = {}
+        self.replay_pkl = "replay.pkl"
+        with open(self.replay_pkl, "wb") as f:
```

Comment:
Similar question -- why is the buffer being written to disk exactly?

Reply:
Because it's easy :P In the future the buffer will be of type Batch and won't be in disk.